### PR TITLE
Eliminate widget-settings-ui-components dependency

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Eliminate widget-settings-ui-components bower dependency
Remove image widget settings UI
Copy widget-button-toolbar and position-setting directives to Apps
Add missing angular-local-storage dependency

## Motivation and Context
Weekly Improvement

## How Has This Been Tested?
Tested locally and on stage-1

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by ezequielc** [comment]
@alex-deaconu There are no unit tests for the added directives in the source repository.
Given this is a weekly improvement card, I don't want it to take much longer. If good with you, we can add unit tests on a later moment. Thanks

File: bower.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "angular-bootstrap": "0.13.1",
     "angular-bootstrap-colorpicker": "~3.0.19",
     "angular-cookies": "~1.7.0",
+    "angular-local-storage": "^0.7.1",
```
Comment (alex-deaconu) [inline bower.json]: @ezequielc to confirm, this was something that was imported by `widget-settings-ui-components`? Thanks.

Comment (ezequielc) [reply bower.json]: @alex-deaconu It was imported by one of `widget-settings-ui-components` dependencies. I could not pinpoint which one exactly, but it started to throw an error once I removed `widget-settings-ui-components`, so I added it to the list of Apps dependencies. Thanks

Comment (alex-deaconu) [reply bower.json]: @ezequielc makes sense, thanks for the clarification!

**Pull request merged**